src/hardware/serialport/serialport.cpp

```diff
@@ -1289,6 +1289,14 @@ bool CSerial::Putchar(uint8_t data, bool wait_dsr, bool wait_cts, uint32_t timeo
 	return true;
 }
 
+uint32_t CSerial::GetPortBaudRate() const {
+	if (baud_divider == 0) {
+		return 115200;
```

Comment:
This is the max baud rate, right? Maybe extract it to some common header as a named constant as it's being used in the other files you touched too.